django/db/models/expressions.py

```diff
@@ -522,6 +526,24 @@ def copy(self):
         copy.extra = self.extra.copy()
         return copy
 
+    @property
+    def lhs(self):
+        """
+        This property exists for compatibility with the previously
```

Comment:
Agreed this is a useful shortcut when writing "transforms", and it feels natural with the `Lookup` api